plugins/zen-bit/includes/class-zen-bit-normalizer.php

```
@@ -142,114 +114,110 @@ private static function normalize_location(array $raw): array
-    private static function build_lineup(array $raw): array
-    {
-        $out = [];
-        if (!empty($raw['lineup']) && is_array($raw['lineup'])) {
-            foreach ($raw['lineup'] as $artist) {
-                $out[] = self::safe_text((string) $artist, 120);
-            }
-        }
-        return $out;
+        $ts = strtotime($datetime);
+        return $ts ? date('c', $ts) : $datetime;
     }
```

PHPMD (2.15.0)

[error] The method parse_iso is not named in camelCase. (undefined)

(CamelCaseMethodName)